client/pokemon/pokemon-full.view.html

```diff
@@ -171,6 +171,7 @@ <h1 class="summary-toolbar">{{pokemon.parsedNickname}}</h1>
                   <span flex>Stat</span>
                   <span flex>Base</span>
                   <span flex>IVs</span>
+                  <span flex ng-if="pokemon.gen === 7">H/T</span>
```

Comment:
Fwiw, we agreed in the IRC that this was fine with just a titletext. I shall add that.